scrunch/mutable_dataset.py

```diff
@@ -0,0 +1,892 @@
+import collections
+import datetime
+import re
+import six
+import pycrunch
+from pycrunch.importing import Importer
+from pycrunch.shoji import wait_progress
+
+from scrunch.expressions import parse_expr, process_expr
+from scrunch.helpers import abs_url, case_expr, subvar_alias
+from scrunch.exceptions import InvalidDatasetTypeError
+from scrunch.datasets import (BaseDataset, _get_connection, _get_dataset,
+                              Variable, CrunchBox)
+from scrunch.subentity import Filter
+from scrunch.variables import (combinations_from_map, combine_categories_expr,
+                               combine_responses_expr, responses_from_map)
+
+_VARIABLE_PAYLOAD_TMPL = {
```

Comment:
All usages of this template always overwrite the name/description/alias. This is yet another use of that `shoji_entity` helper mentioned above and there will be no need for this constant.